web/public/scripts/main.js

```diff
@@ -15,60 +15,118 @@
  */
 'use strict';
 
+import { initializeApp } from 'https://cdn.skypack.dev/firebase@9.0.0-beta.2/app';
```

Comment:
🤯  is there any way to do this without `skypack.dev`?

Reply:
I don't think so, unless we add a build step. There are other disadvantages to an esm import like this, like losing tree shaking. This PR is more of a proof of concept to show we can get the codelab working as-is with the modular SDK, but I'm hoping to get together a PR soon that does have a build step

Reply:
I think the modular SDK will be hosted on our CDN, so this approach works.

Reply:
Ended up deciding on a build step so that we get tree-shaking (and therefore smaller bundles)